pkg/api/handlers/compat/images_build.go

```diff
@@ -205,9 +205,15 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		isolation := buildah.IsolationDefault
 	*/
 	if utils.IsLibpodRequest(r) {
-		//		isolation = buildah.Isolation(query.Isolation)
+		// isolation = parseLibPodIsolation(query.Isolation)
```

Comment:
is this comment of any value?

Reply:
If I understand it correctly the comment should be removed at some point see TODO above.

Reply:
Yes I have a PR that is attempting to remove it.
I think we just merge this and I will continue to rework my PRs